tsa/client.go

```diff
@@ -405,19 +405,34 @@ func (client *Client) checkHostKey(hostname string, remote net.Addr, remoteKey s
 	return errors.New("remote host public key mismatch")
 }
 
+
 func (client *Client) keepAlive(ctx context.Context, sshClient *ssh.Client, tcpConn *net.TCPConn) {
 	logger := lagerctx.WithSession(ctx, "keepalive")
-
+	keepAliveRequestTimeout := 5 * time.Second
 	kas := time.NewTicker(5 * time.Second)
 
 	for {
 		// ignore reply; server may just not have handled it, since there's no
 		// standard keepalive request name
```

Comment:
nit: this comment should be moved to be adjacent to the `SendRequest`

Reply:
sorry, intended this to be a draft as I was still making changes. Its ready to be reviewed now.